### DISCUSSION TITLE
Github Copilot for students

### DISCUSSION THREAD

**Discussion by Mahadkz** [community/community#56828] [GitHub Education]
### Hi everyone,

Hey I just got approved by GitHub education and got approved for global campus. I was supposed to get copilot along with it as well but  I haven't gotten any. For some it worked after couple of hours but for me its been about 20 hours.... any solutions??

**Comment by samus-aran** [comment]
Hi @Mahadkz 👋 

Apologies for the delayed response from the team. I'll be closing this discussion due to lack of recent activity. If you still are having trouble utilising GitHub Global Campus Benefits please feel free to re-open and we'll be happy to assist!
